src/assets/accessibilityStrings.ts

```diff
@@ -47,6 +47,8 @@ const ACCESSIBILITY_STRINGS = {
   questionTitle: 'Vraag pagina',
 
   failedToFetchQuestionnaire: 'Kon vragenlijst niet ophalen',
+  failedToFetchDeletedQuestionaire: 'Vragenlijst bestaat niet meer of verkeerde code',
```

Comment:
```suggestion
  failedToFetchDeletedQuestionaire: 'De vragenlijst bestaat niet meer of er is een verkeerde code meegegeven',
```

Is iets langer, maar wel netter